frontend/src/features/apartment/ApartmentSalesPage.tsx

```diff
@@ -0,0 +1,296 @@
+                ? apartment.prices.maximum_prices.confirmed?.maximum_price / apartment.surface_area
+                : 0,
+            debtFreeMaxPrice: apartment.prices.debt_free_purchase_price,
+            loanAmount: apartment.prices.primary_loan_amount,
+            index: data ? data.index : maxPriceQuery?.index,
+        });
+        setIsModalVisible(true);
+    };
+    console.log(apartment, maxPriceQuery, maxPriceError);
+    return (
+        <>
+            <div className="field-sets">
+                <Fieldset heading="Kaupan tiedot">
+                    <div className="row">
+                        <FormInputField
+                            inputType="date"
+                            label="Ilmoituspäivämäärä"
+                            fieldPath="announcementDate"
+                            formData={formData}
+                            setFormData={setFormData}
+                            error={error}
+                        />
+                        <FormInputField
+                            inputType="date"
+                            label="Kauppakirjan päivämäärä"
+                            fieldPath="dealDate"
+                            formData={formData}
+                            setFormData={setFormData}
+                            error={error}
+                        />
+                    </div>
+                    <div className="row">
+                        <div className={isTooHighPrice ? "input-field--invalid" : ""}>
+                            <FormInputField
+                                inputType="number"
+                                label="Kauppahinta"
+                                unit="€"
+                                fractionDigits={2}
+                                fieldPath="price"
+                                formData={formData}
+                                setFormData={setFormData}
+                                error={error}
+                            />
+                            <div className="error-message error-text">
+                                <span style={{display: isTooHighPrice ? "block" : "none"}}>
+                                    <IconAlertCircleFill />
+                                    Kauppahinta ylittää enimmäishinnan!
+                                </span>
+                            </div>
+                        </div>
+                        <FormInputField
+                            inputType="number"
+                            label="Osuus yhtiön lainoista"
+                            unit="€"
+                            fractionDigits={2}
+                            fieldPath="apartment_share_of_housing_company_loans"
+                            formData={formData}
+                            setFormData={setFormData}
+                            error={error}
+                        />
+                    </div>
+                    <Checkbox
```

Comment:
Should we instead have this?
```
<FormInputField
    inputType="checkbox"
    ...
/>
```

Reply:
Possibly, but I feel this is a bit out of the scope of this **mockup**. When we do the validation I think it'll be necessary to rethink the `FormInput..`s anyway.